### PR TITLE
fix some spelling mistakes

### PR DESCRIPTION
**Pull request by CKchen0726**
fix some spelling mistakes

### CODE REVIEW

File: content/en/blog/_posts/2016-09-00-Kubernetes-1-4-Making-It-Easy-To-Run-On-Kuberentes-Anywhere.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: " Kubernetes 1.4: Making it easy to run on Kubernetes anywhere "
 date: 2016-09-26
 slug: kubernetes-1.4-making-it-easy-to-run-on-kuberentes-anywhere
-url: /blog/2016/09/Kubernetes-1-4-Making-It-Easy-To-Run-On-Kuberentes-Anywhere
+url: /blog/2016/09/Kubernetes-1-4-Making-It-Easy-To-Run-On-Kubernetes-Anywhere
```
Comment (tengqm) [inline content/en/blog/_posts/2016-09-00-Kubernetes-1-4-Making-It-Easy-To-Run-On-Kuberentes-Anywhere.md:5]: Don't do this.

- the file name is there, so the url follows its wrong spelling;
- we don't fix blogs more than 1 year old.

File: content/es/docs/contribute/start.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Si estas buscando información sobre cómo comenzar a contribuir a los repositor
 
 ## Lo básico sobre nuestra documentación
 
-La documentación de Kuberentes esta escrita usando Markdown, procesada y
+La documentación de Kubernetes esta escrita usando Markdown, procesada y
```
Comment (tengqm) [inline content/es/docs/contribute/start.md:21]: I won't touch other languages unless I speak that language.
Who knows if in Spanish "kubernetes" has a different spelling?

**Review by tengqm — CHANGES_REQUESTED** [review]
Please submit PRs separately for each language because not all languages are reviewed by the same team.

File: content/es/docs/contribute/start.md (hunk 2 above)
Comment (CKchen0726) [reply content/es/docs/contribute/start.md:21]: There are many string "Kubernetes" in this file, "Kubernetes" is a proper noun, so I fix the wrong string.

Eg.
## Envía un caso de estudio

Un caso de estudio destaca como organizaciones están usando Kubernetes para resolver problemas del mundo real. Estos se escriben en colaboración con el equipo de marketing de Kubernetes que está dirigido por la {{< glossary_tooltip text="CNCF" term_id="cncf" >}}.

Comment (tengqm) [reply content/es/docs/contribute/start.md:21]: https://kubernetes.io/docs/contribute/localization/#help-an-existing-localization

**Pull request merged**